readme-vars.yml

```diff
@@ -51,7 +51,7 @@ optional_block_1: false
 # application setup block
 app_setup_block_enabled: true
 app_setup_block: |
-  Initial setup is done via http://<docker host ip>:<mapped http port> ie, http://192.168.1.100:8080.
+  Initial setup is done via `http://<docker host ip>:<mapped http port>1 ie, http://192.168.1.100:8080.
```

Comment:
Another 1

Reply:
damn son, i am bad... but i have a fix.
![image](https://github.com/linuxserver/docker-sabnzbd/assets/40674481/9cd89b3b-5328-4faa-a64f-e9bb1946ab5e)

if you see another 1 though.. i just give up.